docs/source/install/prerequisites.rst

```diff
@@ -108,6 +108,12 @@ To start the ``postgres`` database server, execute:
 
     brew services start postgresql
 
+Or, if you don't want/need a background service you can just run:
```

Comment:
Thanks for the suggestion. 
Since this is something that would need to be added to installation sections for all operating systems (also ubuntu, etc.) and all services, I wonder whether there is a way to do this without bloating the instructions for all sections.

The best thing would actually be to add a section for installation via conda, where it is anyhow the user's responsibility to start/stop the services.
Just opened an issue for that https://github.com/aiidateam/aiida-core/issues/3761